lib/tailwind_formatter.ex

```diff
@@ -43,37 +43,17 @@ defmodule TailwindFormatter do
       |> Inspect.Algebra.format(:infinity)
       |> IO.iodata_to_binary()
 
-    sorted_classes =
-      if String.contains?(sorted_classes, "\""),
-        do: trim_classes(sorted_classes),
-        else: sorted_classes
-
     String.replace(contents, expr_class, sorted_classes)
   end
 
-  defp trim_classes(classes) do
-    classes
-    |> String.trim("\"")
-    |> String.trim(" ")
-    |> then(fn str -> "\"#{str}\"" end)
-  end
-
```

Comment:
This PR started out as an investigation into #32 and I found this is what's wrapping a class list with double quotes. I decided to remove this and handle leading and trailing space somewhere else because I imagine it would not work well with #20.